src/WebJobs.Script/Description/DotNet/DotNetScriptTypes.cs

```diff
@@ -5,9 +5,9 @@ namespace Microsoft.Azure.WebJobs.Script.Description
 {
     public class DotNetScriptTypes
     {
-        public const string CSharp = "CSharp";
-        public const string FSharp = "FSharp";
-        public const string DotNetAssembly = "DotNetAssembly";
-        public const string RawDotNetAssembly = "RawDotNetAssembly";
+        public const string CSharp = "csharp";
```

Comment:
With all comparisons being case insensitive, is the casing change here needed? We log this data and this change may impact queries/groupings

Reply:
Reverted this change. Changed to lower case to be consistent with "language" coming from worker.config.json files.